include/color_spinor_field.h

```diff
@@ -96,6 +99,11 @@ namespace quda {
 
     QudaPCType pc_type; // used to select preconditioning method in DWF
 
+    QudaParity suggested_parity; // used to specify whether a single parity field is even/odd
```

Comment:
This should use doxygen markup.

Reply:
Done, using this as a reference: https://www-numi.fnal.gov/offline_software/srt_public_context/WebDocs/doxygen-howto.html